#23181339538 Users should be able to like/unlike an accommodation facility

Tsa2341 opened the pull request:
### what does this Pull Request do?

this Pull Request implements like and unlike accommodations

### description of tasks to be completed

 - user should be able to like an accommodation
 - user should be able to unlike an accommodation

### how should this Pull Request be tested?

 - remottelly: **https://staging-bare-feature-18-yjebiz.herokuapp.com/api-docs**
 - locally: clone this repo and checkout **feature/#23181339538-like-and-unlike-accommodation**  branch
 - add all environment variables and run npm run dev after running npm run dbmigrate:all to seed data
 - checkout this api to login
 
            - **POST**  http://localhost:5000/api/v1/users/login

 - checkout this API to get all accommodations
 
             - **GET**  http://localhost:5000/api/v1/accommodations

- get the token and authenticate with a bearer and then hit these APIs to test the like and unlike endpoints

           - **POST**   http://localhost:5000/api/v1/accommodations/:accommodationId/like
           - **POST**  http://localhost:5000/api/v1/accommodations/:accommodationId/unlike
       

- Worked also on the get trip api where i merged the /trip/ and /trips/manager/ into one /trips/ route

            - **GET** http://localhost:5000/api/v1/trips

### What are the relevant pivotal tracker stories?

 - [#181339538](https://www.pivotaltracker.com/story/show/181339538)

#### Screenshots (if appropriate)

![Screenshot (35)](https://user-images.githubusercontent.com/83913153/159643307-98da532b-4930-44da-a9ff-e85f33f3945f.png)

Code-250 commented on tests/password-reset.test.js:
```diff
@@ -29,17 +29,19 @@ describe('TESTING PASSWORD RESET', () => {
-  });
+  // it('should Send password reset link', (done) => {
+  //   chai
+  //     .request(app)
```
please do not comment other peoples tests

Code-250 reviewed (changes requested):
please do not comment other people's test

blandri committed:
logout (#27)

rebase commit

test a commit

removed unlike

removed the commented test

updated some test

increased user tests

updated swagger

bug: solved role bug in trip service

bug: solved role bug in trip service

ncutixavier commented:
### Comments

- Try to raise your test coverage to be above 85%
